dessia_common/schemas/core.py

```diff
@@ -1033,6 +1033,65 @@ def check_list(self) -> CheckList:
         return CheckList([])
 
 
+class AttributeTypeProperty(TypingProperty):
+    """
+    Schema class for AttributeType and ClassAttributeType type hints.
+
+    A specifically instantiated AttributeType validated against this type.
+    """
+
+    def __init__(self, annotation: Type[AttributeType], attribute: str, definition_default: AttributeType = None):
+        super().__init__(annotation=annotation, attribute=attribute, definition_default=definition_default)
+
+        self.class_ = self.args[0]
+        self.class_schema = get_schema(annotation=self.class_, attribute=attribute,
+                                       definition_default=definition_default)
+
+    @classmethod
+    def annotation_from_serialized(cls, serialized: str):
+        """ Deserialize Attribute annotation. Support Class and Instance attributes. """
+        type_ = TypingProperty.type_from_serialized(serialized)
+        if type_ == "AttributeType":
+            return AttributeType[TypingProperty._args_from_serialized(serialized)]
+        return ClassAttributeType[TypingProperty._args_from_serialized(serialized)]
+
+    def to_dict(self, title: str = "", editable: bool = False, description: str = ""):
+        """ Write AttributeType as a Dict. """
+        chunk = super().to_dict(title=title, editable=editable, description=description)
+        is_class_attribute = self.origin is ClassAttributeType
+        chunk.update({
+            'type': 'object', 'is_attribute': True, 'classattribute_': is_class_attribute,
```

Comment:
The key in the dictionnary should be changed from `classattribute_` to `is_class_attribute`, as well

Reply:
Done